src/pages/generate/pages/result/ResultPage.tsx

```
@@ -105,83 +155,177 @@ const ResultPage = () => {
+                            ? styles.tagButtonSelected
+                            : ''
+                        }`}
+                        onClick={() => handleFactorClick(factor.id)}
+                      >
+                        {factor.text}
+                      </button>
+                    ))}
+                  </div>
+                </div>
+              )}
+            {currentLikeState === 'dislike' &&
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)